Assets/Scripts/Interactable/Interaction.cs

```diff
@@ -1,26 +1,28 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
 public class Interaction : MonoBehaviour
 
-    {
-    
+{
     public Transform detectionPoint;
 
     private const float detectionRadius = 0.5f;
 
     public LayerMask detectionLayer;
 
     public GameObject detectedObject;
+
     void Update()
     {
-        if(DetectObject())
+        if (DetectObject())
         {
-            if(InteractInput())
+            //check if pause menu is currently closed
+            if (!PauseMenu.menuOpen)
             {
-                Debug.Log("Interact");
-                detectedObject.GetComponent<Item>().Interact();
+                if (InteractInput())
```

Comment:
Just a question: Have you ever heard of something called the _logical AND_?
If you'd use that, this method would reduce to a 5-liner.

Reply:
> Just a question: Have you ever heard of something called the _logical AND_? If you'd use that, this method would reduce to a 5-liner.

Yes i know 😅 i only extended the Script in the style it already was. I can change it later tonight